feature-libs/checkout/occ/adapters/occ-checkout-delivery.adapter.ts

```diff
@@ -24,9 +24,39 @@ export class OccCheckoutDeliveryAdapter implements CheckoutDeliveryAdapter {
     protected converter: ConverterService
   ) {}
 
-  protected getCartEndpoint(userId: string): string {
-    const cartEndpoint = 'users/' + userId + '/carts/';
-    return this.occEndpoints.getEndpoint(cartEndpoint);
+  protected deliveryAddressesEndpoint(userId: string, cartId: string): string {
+    return this.occEndpoints.getUrl('deliveryAddresses', { userId, cartId });
+  }
+
+  protected setDeliveryAddressEndpoint(
```

Comment:
I think all these methods should start with `get`. This one is especially confusing as the name suggests that this methods sets delivery address endpoint. and not only gives you "setDeliveryAddress" endpoint